pkg/config/config.go

```diff
@@ -203,18 +203,20 @@ func (c *Config) EditConfig() error {
 
 // PrintConfig outputs the contents of the configuration file.
 func (c *Config) PrintConfig() error {
-	if c.Profile.ProfileName == "default" {
+	profileName := c.Profile.ProfileName
```

Comment:
This is a holdover from some other code changes and isn't important to the PR itself.